governance/README.md

```diff
@@ -102,3 +102,21 @@ yarn hardhat gov:submit --proposal proposals/<your-proposal>.js --network mainne
 ```
 
 4. Head to [Tally](https://www.withtally.com/governance/unlock) to see your proposal. NB: this may take some time as it requires X block confirmations
+
+## Uniswap
+
+### Create a Uniswap V3 Native/UDT pool
+
+Edit the pool fee directly in the script
+
+```
+yarn run scripts/uniswap/createPool.js
+```
+
+### Add liquidity a Uniswap V3 Native/UDT pool
+
+Edit directly the amounts and prices in the script
```

Comment:
Ha! I may need to do it from the hardware wallet though... so can't use a script. How "difficult" is it?

Reply:
Add a mention that it is also possible to use the uniswap UI? ideally with a quick "how to" ?